src/Tests/Microsoft.DotNet.MSBuildSdkResolver.Tests/Microsoft.DotNet.MSBuildSdkResolver.Tests.csproj

```diff
@@ -13,7 +13,11 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Microsoft.DotNet.TemplateLocator\Microsoft.DotNet.TemplateLocator.csproj" />
     <ProjectReference Include="..\..\Resolvers\Microsoft.DotNet.MSBuildSdkResolver\Microsoft.DotNet.MSBuildSdkResolver.csproj" />
-    <ProjectReference Include="..\..\Resolvers\Microsoft.DotNet.SdkResolver\Microsoft.DotNet.SdkResolver.csproj" />
+
+    <!-- Microsoft.DotNet.MSBuildSdkResolver compiles source of Microsoft.DotNet.SdkResolver to reduce dll load. There are duplicated types-->
+    <ProjectReference Include="..\..\Resolvers\Microsoft.DotNet.SdkResolver\Microsoft.DotNet.SdkResolver.csproj">
```

Comment:
Do we need to reference the DLLs from both types?  Should we just reference the MSBuildSdkResolver project and use the types from it?

Reply:
I need to reuse the setup of testing msbuildsdkresolver to test template locator's finding VS version function. It is too much to duplicate, so i just added the template locator test in msbuildsdkresolver tests. And a type from template locator's dependency chain is used in the test. So i need Microsoft.DotNet.SdkResolver too